pkg/cli/etcdsnapshot/etcd_snapshot.go

```diff
@@ -89,7 +89,7 @@ func run(app *cli.Context, cfg *cmds.Server) error {
 		return err
 	}
 	if !initialized {
-		return errors.New("managed etcd database has not been initialized")
+		return fmt.Errorf("etcd database in %s was not initialized", dataDir)
```

Comment:
The past tense 'was' makes it sound like it tried and failed, maybe just say straight up that we couldn't find it where we expected?
```suggestion
		return fmt.Errorf("no etcd database found in %s", dataDir)
```